extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -1038,11 +1038,39 @@ export class AmpLightboxGallery extends AMP.BaseElement {
    *  -Esc will close the lightbox.
    * @private
    */
-  handleKeyboardEvents_(event) {
-    const code = event.keyCode;
-    if (code == KeyCodes.ESCAPE) {
-      this.close_();
+  onKeyDown_(event) {
+    if (!this.isActive_) {
+      return;
+    }
+    const {keyCode} = event;
+    switch (keyCode) {
+      case KeyCodes.ESCAPE:
+        this.close_();
+        return;
+      case KeyCodes.LEFT_ARROW:
+        this.maybeSlideCarousel_(/*Prev*/ -1);
```

Comment:
I like this better than my proposal!

nit: the comment should indicate the argument, which is the same in both cases (`direction`).